src/engraving/dom/check.cpp

```diff
@@ -130,7 +130,7 @@ Ret MasterScore::sanityCheck()
     if (accumulatedErrors.empty()) {
         return muse::make_ok();
     }
-
+    setHasCorruptedMeasures(true);
```

Comment:
I'm not sure if this one is necessary; if a part score has corrupted measures, but the master score itself not, then `hasCorruptedMeasures` should be `false` for the master score.

`score->sanityCheckLocal()` already sets `hasCorruptedMeasures` to `true` if the master score itself has corrupted measures, so this extra `setHasCorruptedMeasures` in `sanityCheck` seems not necessary.

Reply:
removed